rudder-agent/SOURCES/uuidgen

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 # uuidgen doesn't exist on AIX, so provide a compatible shell function
+# This is not compatible with Linux systems.
 
```

Comment:
I would prefer putting all the logic in a single place (this script, but it can be renamed) and  call it from different places, we currently have rudder agent check, rudder agent factory-reset, the postinstall script, and probably others.

So i would prefer this script to handle all the possible cases.